spark/src/main/scala/org/apache/comet/rules/CometExecRule.scala

```diff
@@ -548,19 +385,27 @@ case class CometExecRule(session: SparkSession) extends Rule[SparkPlan] {
           s
         }
 
-      case op: LocalTableScanExec =>
-        if (CometConf.COMET_EXEC_LOCAL_TABLE_SCAN_ENABLED.get(conf)) {
-          operator2Proto(op)
-            .map { nativeOp =>
-              val cometOp = CometLocalTableScanExec(op, op.rows, op.output)
-              CometScanWrapper(nativeOp, cometOp)
+      case op =>
+        // check if this is a fully native operator
+        cometNativeExecHandlers.get(op.getClass) match {
+          case Some(_handler) =>
```

Comment:
_ prefix is usually for unused variables, right? I guess you have a `handler` below, so this helps disambiguate, but _ still confused me at first.

Reply:
Fixed!